src/views/MyEventsPage/index.js

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from "react";
+
+import { Row, Col, Container } from "react-bootstrap";
+
+import DashboardSidebar from "../../components/DashboardSidebar";
+import DashboardCard from "../../components/DashboardCard";
+import GradientBackground from "../../components/GradientBackground";
+import LoadingSpinner from "../../components/LoadingSpinner";
+import MyEventsCard from "../../components/MyEventsCard";
+
+import styles from "./styles.module.css";
+
+import {
+  getJwt,
+  getEmailFromJwt,
+  getNameFromJwt,
+  getRoleFromJwt,
+} from "../../utils/Cognito/index.js";
+import { getApplication } from "../../utils/API/index.js";
+
+const applicationDeadline = new Date(
+  process.env.REACT_APP_APPLICATION_DEADLINE
+);
+const today = new Date();
+
+export default class MyEventsPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loadComplete: false,
+      points: 0,
+      events: {},
+    };
+
+    if (getJwt()) {
+      this.state.email = getEmailFromJwt();
+    } else {
+      this.props.history.push({
+        pathname: "/login",
+      });
+    }
+  }
+
+  componentDidMount() {
+      // call API here to get actual points and events details
+      this.setState({
+        loadComplete: true,
+        points: 2,
```

Comment:
I totally forgot to mention this - Corporate decided that all events are worth 1 point, so we can calculate points just by counting the number of events that the user has attended!

I think that Javascript should have an easy list manipulation function to count objects in a list with `attended == true`?

Reply:
RESOLVED: Put a .filter and .length function there now, thank you for the update and for looking it over